doc/classes/EditorFileDialog.xml

```diff
@@ -19,6 +19,14 @@
 				For example, a [param filter] of [code]"*.tscn, *.scn"[/code] and a [param description] of [code]"Scenes"[/code] results in filter text "Scenes (*.tscn, *.scn)".
 			</description>
 		</method>
+		<method name="add_side_menu">
+			<return type="void" />
+			<param index="0" name="menu" type="Control" />
+			<param index="1" name="title" type="String" default="&quot;Settings:&quot;" />
```

Comment:
Ah, missed this. It's going to fail the doctool check, you need to update the default in the docs as well :)